docs/pages/blog/introducing-mui-base.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Introducing MUI Base: the headless alternative to Material UI'
+description: The MUI Base component library gives you complete control over the look and feel of your app.
+date: 2022-09-05T00:00:00.000Z
+authors: ['michaldudak', 'samuelsycamore']
+tags: ['News', 'MUI Core']
+card: true
+---
+
+<img src="/static/blog/introducing-mui-base/hero-image.png" style="width: 692px; aspect-ratio: 132/61; margin-bottom: 24px;" alt="Demo components built with MUI Base, a newly introduced library of unstyled components and hooks" />
```

Comment:
One thing that I experienced while reading the blog post was: OK, I'm sold, where do I find these components? Developers have to find this link: 

<img width="626" alt="Screenshot 2022-09-05 at 17 08 02" src="https://user-images.githubusercontent.com/3165635/188478574-90d5d6eb-25a1-40a7-bd67-540afbe008df.png">

This got me wondering if would make sense to turn this image into a link, as part of making it easier.

```suggestion
<a href="/base/getting-started/installation/">
<img src="/static/blog/introducing-mui-base/hero-image.png" style="width: 692px; aspect-ratio: 132/61; margin-bottom: 24px;" alt="Demo components built with MUI Base, a newly introduced library of unstyled components and hooks" />
</a>
```

Reply:
IMO having a hero image link to the docs isn't something I'd expect as a reader. I'll add a link near the bottom of the page instead.

Reply:
Yeah, I'm not sure that I would expect it, but I did catch myself trying it out as "maybe" 😁.
Tailwind's blog has most of its image with a link, e.g. https://tailwindcss.com/blog/2022-08-17-tailwind-framer-motion-template-and-tailwind-jobs#tailwind-jobs, same goes for Bootstrap, e.g. https://blog.getbootstrap.com/2022/09/07/bootstrap-5-2-1/.